docs/kubeadm.md

```diff
@@ -68,13 +68,10 @@ For each host in turn:
 * SSH into the machine and become `root` if you are not already (for example, run `sudo su -`).
 * If the machine is running Ubuntu 16.04, run:
 
-      # curl -sSL https://get.docker.com/ | sh
-      # apt-get install -y socat
+      # apt-get install -y docker.io socat
       # curl -s -L \
-        "https://www.dropbox.com/s/shhs46bzhex7dxo/debs-9b4337.txz?dl=1" | tar xJv
+        "https://www.dropbox.com/s/tso6dc7b94ch2sk/debs-5ab576.txz?dl=1" | tar xJv
```

Comment:
Yes, this will be replaced by the packages that @mikedanese is going to bake as soon as https://github.com/kubernetes/kubernetes/pull/33262 and https://github.com/kubernetes/kubernetes/pull/32203 land. Same for RPMs, they will be hosted on packages.cloud.google.com.